### ISSUE TITLE
Include all metadata in separate file with zip download

### ISSUE THREAD

**Issue by lauradryad** [datadryad/dryad-product-roadmap#3398]
**Problem description, and links to relevant helpdesk tickets:**

When a user uses the Download data button, they receive all data files and the README, but not a copy of the landing page metadata (abstract, authors, RW, etc.), separating that information from users after download. 

**Describe the solution you'd like:**

Once a dataset is published, the zip file generated with the Download dataset button should also include a file (maybe markdown) of all the dataset metadata so that it is a complete package.

**Comment by jleighherzog** [comment]
Wait. Including the metadata will not support a double-blind peer review process. Need to reconsider.

**Comment by lauradryad** [comment]
This is after publication, so the PPR process is in the past at this point.

**Comment by jleighherzog** [comment]
And, that's also mentioned in the description that you provided above. Monday is not my day.

**Comment by ahamelers** [comment]
All the stuff that is not the files (including the README) seems like the only reason people would have to visit our landing pages (which is what increases the views for the datasets). It may not make sense to deliver everything via download!

**Comment by ahamelers** [comment]
@jleighherzog yes but once anyone has the files, they can be passed on to other people, who would not need to visit our site at all—unless they're curious about seeing more. I'm hesitant to take away from the "more"!
